phd-tests/tests/src/crucible/smoke.rs

```diff
@@ -88,8 +88,9 @@ async fn shutdown_persistence_test(ctx: &Framework) {
 }
 
 #[phd_testcase]
-async fn vcr_replace_test(ctx: &Framework) {
-    let mut config = ctx.vm_config_builder("crucible_vcr_replace_test");
+async fn vcr_replace_during_start_test(ctx: &Framework) {
```

Comment:
Should we also have a test for stopping while starting, to exercise the `buffered_external` queue?

Reply:
We absolutely should! Added one in bfb8b07, which immediately found several bugs that are also fixed in that commit.